datapolicies.md

```diff
@@ -2,125 +2,91 @@
 layout: base
 permalink: /datapolicies/
 ---
-# Data Policies -- DRAFT
+# Data Policies
 
-This page contains a **DRAFT** data policies proposal.
-
-This page describes the disclosure levels for observational (generation
-and environmental) and forecast data submitted to the framework, as well
-as data derived from the process of comparing forecast fields to
-observations (e.g. summary statistics).
+This page summarizes the data sharing and privacy policies of the
+Solar Forecast Arbiter.
 
 An **organization** is an entity that owns data or obtains license to submit
 data to the framework. A **user** is an individual working for an organization
 that performs the tasks of submitting data to the framework, granting other
 users or organizations rights to view data, and downloads data from the
 framework.
 
-
-Guiding Principles
-------------------
-
-The following principles guide our data policies.
-
-1. Organizations retain ownership of the data they upload to the framework.
-2. Users upload data to the framework on behalf of organizations.
-2. Users have complete control over how their organization's data may be
-   accessed by other users within organizations.
-2. Users may delete their organization's data from the framework at any time.
-2. The framework will not sell, donate, or otherwise disclose the data that it
-   controls without express written consent of the data owner.
-2. All non-public data will be securely deleted by the conclusion of the DOE
-   funding period (June 30, 2021) unless express written permission to
-   retain it is obtained.
-
-
-Contributing data
------------------
+The [Data Use Agreement](/assets/45864_UAZ_Solar_Forecast_Arbiter_Final.pdf)
+(DUA) is a **non-negotiable** legal document that all parties are bound to.
+The DUA must be signed by an authorized representative
+of an organization before its employees will be allowed to upload data,
+view data contributed by other users, or generate summary statistics.
+
+The data policies can be summarized as:
+
+* Signing the DUA does **not** obligate an organization to upload data,
+  nor does it obligate an organization to share uploaded data.
+* Organizations retain ownership of the data they upload to the framework.
+* Users upload data to the framework on behalf of organizations.
+* Users have complete control over how their organization's data may be
+  accessed by other users.
+* Users may delete their organization's data from the Arbiter at any time.
```

Comment:
admin or user w/ permission, just so people don't think it's willy nilly control within the org